libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -0,0 +1,282 @@
+    get disabled(): boolean {
+        return this._disabled;
+    }
+    set disabled(value) {
+        if (this._disabled !== value) {
+            this._disabled = coerceBooleanProperty(value);
+            this._disableRadioButtons();
+        }
+    }
+
+    /** Child radio buttons. */
+    @ContentChildren(
+        forwardRef(() => RadioButtonComponent),
+        { descendants: true }
+    )
+    radioButtons: QueryList<RadioButtonComponent>;
+
+    /**
+     * Event emitted when the group value changes.
+     * Change events are only emitted when the value changes due to user interaction with
+     * a radio button (the same behavior as `<input type-"radio">`).
+     */
+    @Output()
+    readonly change: EventEmitter<RadioButtonChange> = new EventEmitter<RadioButtonChange>();
+
+    /** Makes radio buttons in one line */
+    protected _isInline: boolean = false;
+
+    /** Selected value for the radio group. */
+    protected _value: any = null;
+
+    /** The HTML name attribute applied to radio buttons in this group. */
+    protected _name: string = `radio-group-${nextUniqueId++}`;
+
+    /** The currently selected radio button. Should match value. */
+    private _selected: RadioButtonComponent | null = null;
+
+    /** Whether the `value` has been set to its initial value. */
+    private _isInitialized: boolean = false;
+
+    /** Whether the radio group is disabled. */
+    protected _disabled: boolean = false;
+
+    constructor(
+        private _changeDetector: ChangeDetectorRef,
+        @Optional() @Self() public ngControl: NgControl,
+        @Optional() @Self() public ngForm: NgForm
+    ) {
+        super(_changeDetector, ngControl, ngForm);
+    }
+
+    /**
+     * checks selected radio button
+     */
+    _checkSelectedRadioButton() {
+        if (this._selected && !this._selected.checked) {
+            this._selected.inputElement.nativeElement.checked = true;
+        }
+    }
+
+    /**
+     * Initialize properties once content children are available.
+     * This allows us to propagate relevant attributes to associated buttons.
+     */
+    ngAfterContentInit() {
+        // Mark this component as initialized in AfterContentInit
+        // because content will be initialised After this
+
+        this._isInitialized = true;
+        this._initRadioButtons();
+    }
+
+    /**
+     * @hidden
+     * @internal
+     */
+    private _initRadioButtons() {
+        this._updateRadioButtonNames();
+        if (this.radioButtons) {
+            const props = { name: this._name, disabled: this._disabled };
+            this.radioButtons.forEach(button => {
+                Object.assign(button, props);
+                if (button.value === this._value) {
+                    button.inputElement.nativeElement.checked = true;
+                    this._selected = button;
+                }
+                // button.onChange.Pipe(takeUntil(this.destroy$)).subscribe(ev => this._selectedRadioButtonChanged(ev));
```

Comment:
@fkolar , Radio button have not exposed change event. By using Onchange function i am not able to get button change event. please suggest here.